e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -99,28 +99,29 @@ export function interceptPromise(method, path) {
  *   cy.visit(`/dashboard/1`);
  * });
  */
-const cypressWaitAllRecursive = (results, currentCommand, commands) => {
-  return currentCommand.then(result => {
-    results.push(result);
+const cypressWaitAllRecursive = (results, commands) => {
+  const [nextCommand, ...restCommands] = commands;
+  if (!nextCommand) {
+    return results;
+  }
 
-    const [nextCommand, ...rest] = Array.from(commands);
+  return nextCommand.then(result => {
+    results.push(result);
 
     if (nextCommand == null) {
-      return results;
+      return;
     }
```

Comment:
We can remove this condition because a similar one now lives in line 104.

Reply:
Good point.

Reply:
Fixed in b9137d510c71599ec0c6dd0659de0726e356f4da